### PR TITLE
Add practice exercise `variable-length-quantity`

### PR DESCRIPTION
**Pull request by jiegillet**
@lpil you will like this one, it's great to practice bitstrings.

### CODE REVIEW

File: exercises/practice/variable-length-quantity/.meta/example.gleam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+fn encode_one(integer: Int, is_first_call: Bool) -> BitString {
+  let leading_bit = case is_first_call {
+    True -> 0
+    False -> 1
+  }
+
+  let small_bits = integer % 128
+  let small_byte = <<leading_bit:1, small_bits:7>>
```
Comment (jiegillet) [inline exercises/practice/variable-length-quantity/.meta/example.gleam:22]: Here I first tried
`<<leading_bit:1, integer % 128:7>>`
but it didn't work. Then I tried
`<<leading_bit:1, {integer % 128} :7>>`
but the `{}` got stripped away by the formatter.

Is this worth opening an issue or is it already on your radar?

Comment (lpil) [reply exercises/practice/variable-length-quantity/.meta/example.gleam:22]: Could you open an issue please? 🙏

**Review by lpil — APPROVED** [review]
Very nice!!

File: exercises/practice/variable-length-quantity/.meta/example.gleam (hunk 1 above)
Comment (jiegillet) [reply exercises/practice/variable-length-quantity/.meta/example.gleam:22]: Done! https://github.com/gleam-lang/gleam/issues/2045

**Pull request merged**